inc/class-registration.php

```diff
@@ -794,6 +801,11 @@ public function subscribe_fa( $block_content, $block ) {
 			}
 		}
 
+		if ( 'themeisle-blocks/accordion' === $block['blockName'] ) {
```

Comment:
@arinaturcu Here you can make sure it's only loaded when required so based on some attribute checking.

Reply:
Since by default it uses the `angle-down` icon, it will always be required. Should I leave it like it was before, with no icon, until one is selected?

Reply:
Yea. The default one should be the one that we had before. This way we won't be loading Font Awesome unless needed hence a lower footprint for the users.